docs/installation/imgproxy.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Imgproxy
+---
+
+Imgproxy is a standalone server for resizing images. It is optional and you do
+not need it to start Forem. We are currently only it using Forem-cloud's
+application.
+
+## Installation
+
+On macOS, use homebrew with `brew install imgproxy`. For others, please see the
+[official documentation](https://docs.imgproxy.net/#/installation)
```

Comment:
`For other operating systems`

I wonder if this page should be linked in the various installation guides, otherwise when is the user going to hit the "imgproxy" page in its installation workflow?

Reply:
I can clarify this further.